addons/mail/static/src/components/message/message_tests.js

```diff
@@ -93,7 +93,7 @@ QUnit.test('basic rendering', async function (assert) {
     );
     assert.strictEqual(
         messageEl.querySelector(`:scope .o_Message_authorName`).textContent,
-        "Demo User",
+        "Note by Demo User",
```

Comment:
should not change functionally

Reply:
"Note by" is an invisible text, functionally it should be the same

Reply:
Done, changes to tests not required anymore.